### PR TITLE
Add rolebinding to Configure Multiple Schedulers

### PR DESCRIPTION
**Pull request by Kartik494**
Fixes: #37840 

This PR document rolebinding  to configure-multiple-schedulers.

### CODE REVIEW

**Comment by Kartik494** [comment]
/sig scheduling
cc @tengqm

File: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 1]
```diff
@@ -75,6 +75,11 @@ In the above manifest, you use a [KubeSchedulerConfiguration](/docs/reference/sc
 to customize the behavior of your scheduler implementation. This configuration has been passed to
 the `kube-scheduler` during initialization with the `--config` option. The `my-scheduler-config` ConfigMap stores the configuration file. The Pod of the`my-scheduler` Deployment mounts the `my-scheduler-config` ConfigMap as a volume.
 
+Grants a Role  within a specific namespace. Example:
+ ```shell
+    kubectl create rolebinding -n kube-system my-scheduler-extension-apiserver-authentication-reader --role=extension-apiserver-authentication-reader --serviceaccount=kube-system:my-scheduler
```
Comment (sftim) [inline content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:80]:
```suggestion
kubectl create rolebinding -n kube-system my-scheduler-extension-apiserver-authentication-reader --role=extension-apiserver-authentication-reader --serviceaccount=kube-system:my-scheduler
```

What about the ServiceAccount? Where is that created?

Comment (Kartik494) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md]: Done

Comment (Kartik494) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md]: @sftim ServiceAccount is created in kube-system namespace , please see here
https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/#define-a-kubernetes-deployment-for-the-scheduler

Comment (Kartik494) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md]: > $ kubectl get serviceaccount -n kube-system
> NAME                                 SECRETS   AGE
> attachdetach-controller              0         18d
> bootstrap-signer                     0         18d
> calico-kube-controllers              0         18d
> canal                                0         18d
> certificate-controller               0         18d
> clusterrole-aggregation-controller   0         18d
> coredns                              0         18d
> cronjob-controller                   0         18d
> daemon-set-controller                0         18d
> default                              0         18d
> deployment-controller                0         18d
> disruption-controller                0         18d
> endpoint-controller                  0         18d
> endpointslice-controller             0         18d
> endpointslicemirroring-controller    0         18d
> ephemeral-volume-controller          0         18d
> expand-controller                    0         18d
> generic-garbage-collector            0         18d
> horizontal-pod-autoscaler            0         18d
> job-controller                       0         18d
> kube-proxy                           0         18d
> my-scheduler                         0         4m32s

Please find the output here

Comment (Kartik494) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md]: Hi @sftim Could you please let me know if i addressed your query properly , or if any other thing is left.

File: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 2]
```diff
@@ -75,6 +75,11 @@ In the above manifest, you use a [KubeSchedulerConfiguration](/docs/reference/sc
 to customize the behavior of your scheduler implementation. This configuration has been passed to
 the `kube-scheduler` during initialization with the `--config` option. The `my-scheduler-config` ConfigMap stores the configuration file. The Pod of the`my-scheduler` Deployment mounts the `my-scheduler-config` ConfigMap as a volume.
 
+Grants a Role  within a specific namespace. Example:
+ ```shell
+kubectl create rolebinding -n kube-system my-scheduler-extension-apiserver-authentication-reader --role=extension-apiserver-authentication-reader --serviceaccount=kube-system:my-scheduler
+ ```
```
Comment (tengqm) [inline content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:81]:
```suggestion
Grants a Role within a specific namespace. For example:

```shell
kubectl create rolebinding -n kube-system \
    my-scheduler-extension-apiserver-authentication-reader \
    --role=extension-apiserver-authentication-reader \
    --serviceaccount=kube-system:my-scheduler
```
````

Comment (Kartik494) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md]: @tengqm Done PTAL!!

**Comment by divya-mohan0209** [comment]
@kubernetes/sig-scheduling-pr-reviews : PTAL and provide a signoff from tech perspective.

File: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 3]
```diff
@@ -75,6 +75,15 @@ In the above manifest, you use a [KubeSchedulerConfiguration](/docs/reference/sc
 to customize the behavior of your scheduler implementation. This configuration has been passed to
 the `kube-scheduler` during initialization with the `--config` option. The `my-scheduler-config` ConfigMap stores the configuration file. The Pod of the`my-scheduler` Deployment mounts the `my-scheduler-config` ConfigMap as a volume.
 
+Grants a Role within a specific namespace. For example:
```
[the remark this answers is not in the document]
Comment (alculquicondor) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:78]: Actually, why not make it part of this yaml?

https://raw.githubusercontent.com/kubernetes/website/main/content/en/examples/admin/sched/my-scheduler.yaml

[the remark this answers is not in the document]
Comment (Kartik494) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md]: I have done the changes as `Grant a Role within the namespace where your scheduler is running. For example:`  , and sure i will be modified rest asap. Thanks for your suggestions.

[the remark this answers is not in the document]
Comment (alculquicondor) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md]: I think it makes more sense to put it in the yaml, to be consistent with the rest of the role bindings

[the remark this answers is not in the document]
Comment (Kartik494) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md]: Sure i will address the changes.

[the remark this answers is not in the document]
Comment (Kartik494) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md]: Hi @alculquicondor as per your suggestions i have modified the PR and add the content to yaml itself.

[the remark this answers is not in the document]
Comment (Kartik494) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md]: Please find the output as  the yaml successful deployed.

```
 kubectl apply -f shceduler.yaml
serviceaccount/my-scheduler created
clusterrolebinding.rbac.authorization.k8s.io/my-scheduler-as-kube-scheduler created
clusterrolebinding.rbac.authorization.k8s.io/my-scheduler-as-volume-scheduler created
configmap/my-scheduler-config created
rolebinding.rbac.authorization.k8s.io/my-scheduler-extension-apiserver-authentication-reader created
deployment.apps/my-scheduler created
```

**Comment by Kartik494** [comment]
Hi @tengqm i have modified the PR again as requested by @alculquicondor please let me know if this sounds ok to you.

**Pull request merged**